x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/components/package_list_grid.tsx

```diff
@@ -59,18 +79,17 @@ export function PackageListGrid({
       <EuiFlexItem grow={1}>{controlsContent}</EuiFlexItem>
       <EuiFlexItem grow={3}>
         <EuiSearchBar
-          query={searchTerm}
+          query={query as Query}
```

Comment:
Not important, but instead of using an assertion here if query can be null(?), perhaps:
```suggestion
          query={query || undefined}
```